internal/consumer/consumer.go

```diff
@@ -4,11 +4,15 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"go.opentelemetry.io/otel"
 	"os"
 	"os/signal"
 	"syscall"
 
+	"github.com/project-kessel/inventory-api/internal/authz/allow"
+	kessel2 "github.com/project-kessel/inventory-api/internal/authz/kessel"
```

Comment:
nitpicking here but I don't love the `kessel` naming for imports in these files haha

thoughts on renaming this to `authz` or `relations`?

or even renaming kessel to something like `v1` and keep this as `kessel`

Reply:
yeah i mean to revisit this, i dont love it either. it got really hard to find something because we import basically all those terms. I tried changing kessel to v1 so that this could be kessel but got errors, lemme see if i can get around it

Reply:
fixed -- dont know why it gave me such a hard time last time i tried changing it haha